src/components/profiles-modals/DeleteProfileModal.vue

```diff
@@ -0,0 +1,93 @@
+<script lang="ts">
+import { Vue, Component } from 'vue-property-decorator';
+import { ModalCard } from "../all";
+import FileUtils from "../../utils/FileUtils";
+import R2Error from "../../model/errors/R2Error";
+import Profile from "../../model/Profile";
+import FsProvider from "../../providers/generic/file/FsProvider";
+import path from "path";
+
+
+let fs: FsProvider;
+
+@Component({
+    components: {ModalCard}
+})
+export default class DeleteProfileModal extends Vue {
+
+    //TODO: add to vuex store or something to update Profiles list correctly.
```

Comment:
Sounds good to me. There's two ways that *might* work.

1. Have the vuex state contain the profile list, and an action that updates the list in state (i.e. doing what `updateProfileList` does)
2. Just have a getter method that returns the list (i.e. does what `updateProfileList` does, but just returns the list instead of storing it internally)

I think 2) might be the nicer way to do this IF:
- The reactivity works as expected, i.e. the UI updates always when the list on disk updates
- This doesn't become a performance hit (it might, if we end up reading from the disk every time something happens on the UI).

So I would recommend to spent a moment to check if 2) seems feasible, and falling back to 1) if it gives us trouble.